### ISSUE TITLE
[REQUEST]  add attribute __version__

### ISSUE THREAD

**Issue by Salim9304** [willmcgugan/rich#1754]
The rich api changes frequently, requires access to the current version of lib rich (print (rich .__ version__)) from the python interpreter itself.

`AttributeError: module 'rich' has no attribute '__version__'`

**Comment by willmcgugan** [comment]
https://github.com/willmcgugan/rich/issues?q=is%3Aissue+is%3Aclosed++__version__

**Comment by Salim9304** [comment]
> https://github.com/willmcgugan/rich/issues?q=is%3Aissue+is%3Aclosed++__version__

The link is empty. I also could not find information on the __version request.
Can you tell us what you mean? 🤗

@willmcgugan

**Comment by willmcgugan** [comment]
Search issues for __version__

**Comment by Salim9304** [comment]
Solution I found (I'm sorry I wasted my time on this). It's not worth it.

```
 import pkg_resources
 pkg_resources.get_distribution("rich").version

```
9.5.0

I don't want to take this approach (it's slow and resource-intensive). I'm not even sure if it worked on my system, it might not work for others.
Better to duplicate code than importlib_metadata (comments on issues on this topic speak for themselves).
https://github.com/python-poetry/poetry/issues/144
